opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/Xs2aTransactionListConsentInitiate.java

```diff
@@ -46,13 +48,10 @@ protected void doValidate(DelegateExecution execution, TransactionListXs2aContex
     @Override
     protected void doRealExecution(DelegateExecution execution, TransactionListXs2aContext context) {
         ValidatedHeadersBody<ConsentInitiateHeaders, Consents> params = extractor.forExecution(context);
-        Response<ConsentCreationResponse> consentInit = ais.createConsent(
-                params.getHeaders().toHeaders(),
-                params.getBody()
+        errorSink.swallowAuthorizationErrorForLooping(
```

Comment:
We should pass error extractor in this call or to create other `AuthorizationErrorSink` class that parses IBAN issues exclusively

Reply:
done